src/windows/wslaservice/exe/WSLAVirtualMachine.h

```diff
@@ -149,7 +149,6 @@ class DECLSPEC_UUID("0CFC5DC1-B6A7-45FC-8034-3FA9ED73CE30") WSLAVirtualMachine
 
     GUID m_vmId{};
     std::wstring m_vmIdString;
-    wsl::windows::common::helpers::WindowsVersion m_windowsVersion = wsl::windows::common::helpers::GetWindowsVersion();
```

Comment:
There is no reason to have a member variable, this is a global static in the helper that is initialized the first time the version functions are called.